Cargo.toml

```diff
@@ -34,11 +34,11 @@ panic = 'abort'
 rpath = false
 
 [features]
-default = []
+default = ["astroport", "osmosis"]
```

Comment:
delet

Reply:
then Pool becomes empty and a bunch of issues pop up. you need to actually compile it with either one of the features (otherwise there is no functionality) but I didn't find a good way to force compilation with at least one of the features

Reply:
In vault zapper I tried doing:
```rust
// Force selecting either `astroport` or `osmosis` features, or both.
#[cfg(not(any(feature = "astroport", feature = "osmosis")))]
compile_error!("Must select either `astroport` or `osmosis` feature, or both");
```

But, unfortunately `abstractmoney/rust-optimizer` tries to compile one version without features so fails CI.